telemetry/csharp/AwsToolkit.Telemetry.Events.Tests/GeneratedCodeTests.cs

```diff
@@ -217,6 +217,41 @@ public void RecordMetricWithNewTransform()
             Assert.Single(datum.Metadata);
         }
 
+        /// <summary>
```

Comment:
Test to make sure `allowedValues` with spaces converts the key properly and make sure the allowedValue -> value has spaces for now.

Although its recommended to NOT have spaces in the values, the build should not fail and have consistent behavior across environments